### PR TITLE
Add Listenable

### PR DESCRIPTION
**Pull request by MerrickSapsford**
Added Listenable.

## Project URL
https://github.com/MerrickSapsford/Listenable

## Description
Swift object that provides an observable platform for multiple listeners.

## Checklist
- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 or later
- [x] Supports Swift 3
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by lfarah** [comment]
@dkhamsing I'm not sure about this category

**Comment by MerrickSapsford** [comment]
@lfarah maybe more of a utility?

**Comment by victorgama** [comment]
@lfarah @MerrickSapsford Reactive, perhaps?

**Comment by lfarah** [comment]
@merrickp please rebase and move it to `/Reactive`

File: README.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 * [SwiftPriorityQueue](https://github.com/davecom/SwiftPriorityQueue) - A priority queue with a classic binary heap implementation in pure Swift. :large_orange_diamond:
 * [Pencil](https://github.com/naru-jpn/pencil) - Write values to file and read it more easily. :large_orange_diamond:
 * [HeckelDiff](https://github.com/mcudich/HeckelDiff) - A fast Swift diffing library. :large_orange_diamond:
-* [Listenable](https://github.com/MerrickSapsford/Listenable) - Swift object that provides an observable platform. :large_orange_diamond:
+* [Dekoter](https://github.com/artemstepanenko/Dekoter) - `NSCoding`'s counterpart for Swift structs. :large_orange_diamond:
```
Comment (dkhamsing) [inline README.md]: why was this added?

Comment (MerrickSapsford) [inline README.md]: Ended up doing a manual merge with latest, apologies!

**Comment by dkhamsing** [comment]
thx!
